src/pages/index.js

```diff
@@ -137,3 +225,30 @@ editProfileButton.addEventListener("click", () => {
 addCardButton.addEventListener("click", () => {
   addCard.open();
 });
+
+editAvatarButton.addEventListener("click", () => {
+  editAvatar.open();
+});
+
+/* 
+  ┌─────────────────────────────────────────────────────────────────────────┐
+  │ CREATE APIS                                                             │
+  └─────────────────────────────────────────────────────────────────────────┘
+ */
+
+const profileInfo = new Api();
```

Comment:
`new Api()` should be called once to create only **one** api instance. You don't need to do that twice